GAE/src/org/waterforpeople/mapping/app/web/ProcessorServlet.java

```diff
@@ -162,4 +175,18 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
         }
 
     }
+    
+    private Long parseFormID(HttpServletRequest req) {
+        String formID = StringUtils.trim(req.getParameter("formID"));
+        if (StringUtils.isNotBlank(formID)) {
```

Comment:
Not really. This is a new property whose format can be defined now. It's just a matter of taste. If `.*Id` feels more consistent with the other variables, we can easily adapt the code.